src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,145 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { countTodos } from '../../utils/countTodos';
+import { useTodos } from '../Contexts/TodosContext';
+import { useErrorMessage } from '../Contexts/ErrorMessageContext';
+import { USER_ID } from '../../utils/userToken';
+import { useLoadingTodos } from '../Contexts/LoadingTodosContext';
+import { addTodo, updateTodo } from '../../api/todos';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  onWaiting: (tempTodo: Todo | null) => void,
+};
+
+export const NewTodo: React.FC<Props> = ({ onWaiting }) => {
+  const { todos, setTodos } = useTodos();
+  const { setLoadingTodos } = useLoadingTodos();
+
+  const { setErrorMessage, setIsErrorHidden } = useErrorMessage();
+  const [newTitle, setNewTitle] = useState('');
+
+  const onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
```suggestion
  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364